src/sentry/static/sentry/app/views/eventsV2/results.tsx

```diff
@@ -286,4 +282,20 @@ export const Top = styled('div')`
   flex-grow: 0;
 `;
 
-export default withApi(withOrganization(withGlobalSelection(Results)));
+function ResultsContainer(props: Props) {
+  /**
+   * Block `<Results>` from mounting until GSH is ready since there are API
+   * requests being performed on mount.
+   *
+   * Also, we skip loading last used projects if you have multiple projects feature as it
```

Comment:
This seems incomplete.

Reply:
Ooops, updated, does this sound accurate?

```
Also, we skip loading last used projects if you have multiple projects feature as
you no longer need to enforce a project if it is empty. We assume an empty project is
the desired behavior because saved queries can contain a project filter.
```